<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>IssueBook</class>
 <widget class="QMainWindow" name="IssueBook">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>541</width>
    <height>380</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Issue Book</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QWidget{
	background-color: #1C1C1C;
}

QLineEdit{
	background-color: #2F2F2F;
	border-radius: 3px;
	color: white;
	border-bottom: 1px;
	border-style: solid;
	border-color: #979797;
}

QLineEdit:focus{
	border-color: #60CDFF;
}

QPushButton
{
    color: white;
    font-size: 15px;
    background-color: #292929;
    border-radius: 5px;
    font-family: &quot;Noto Sans&quot;
}

QPushButton::hover{ 
    background-color: #2F2F2F;
}

QPushButton:pressed{
    border-width: 1px;
    border-style: solid;
	border-color: white;
}

QLabel
{
    color: white;
	font-size: 16px;
    font-family: &quot;Noto Sans&quot;;l
    border-radius: 5px;
}
</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>30</y>
      <width>111</width>
      <height>28</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Noto Sans</family>
      <pointsize>-1</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Book Name</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>150</y>
      <width>101</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>Username</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>210</y>
      <width>101</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>Issue Date</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>270</y>
      <width>101</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>Comments</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="txtBookName">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>30</y>
      <width>291</width>
      <height>28</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="txtUsername">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>150</y>
      <width>291</width>
      <height>28</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="txtIssueDate">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>210</y>
      <width>331</width>
      <height>28</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="txtComments">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>270</y>
      <width>331</width>
      <height>28</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="searchBook">
    <property name="geometry">
     <rect>
      <x>470</x>
      <y>30</y>
      <width>30</width>
      <height>30</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton
{
    color: white;
    font-size: 15px;
    background-color: #292929;
    border-radius: 5px;
}

QPushButton::hover{ 
    background-color: #2F2F2F;
}

QPushButton:pressed{
    border-width: 1px;
    border-style: solid;
	border-color: white;
}</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="searchUser">
    <property name="geometry">
     <rect>
      <x>470</x>
      <y>150</y>
      <width>31</width>
      <height>30</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton
{
    color: white;
    font-size: 15px;
    background-color: #292929;
    border-radius: 5px;
}

QPushButton::hover{ 
    background-color: #2F2F2F;
}

QPushButton:pressed{
    border-width: 1px;
    border-style: solid;
	border-color: white;
}</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>90</y>
      <width>111</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>Author Name</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="txtAuthorName">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>90</y>
      <width>331</width>
      <height>28</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="issueButton">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>330</y>
      <width>106</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>Issue</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
